### DISCUSSION TITLE
Is `fresh` older than `aleph.js`?

### DISCUSSION THREAD

**Discussion by italodeandra** [alephjs/aleph.js#493] [General]
After Deno announced the release of the first stable version, I remember searching what full stack frameworks people were building at the time.

`aleph.js` seemed to be the only one back then, or at least, the most famous.

A couple years later I have seen two others `ultra` and `fresh`. `fresh` has the same inspiration of `next.js` but it now seems way more popular than `aleph.js`. Why did that happen? Is `fresh` older than `aleph.js`? If you have used both how do you feel about them?

**Comment by ije** [comment]
fresh is using preact and the island arch is super cool i love it, if you want to use react 18 streaming ssr try ultra.
the refactor of aleph.js is to support different frameworks like react/vue/solid, and easy to deploy to Deno Deploy, checkout our examples, currently i working on the the docs, will release the beta version very soon!
